docs/Intro/dashboard.md

```diff
@@ -0,0 +1,47 @@
+---
+sidebar_position: 4
+---
+
+# The Chronicle Dashboard
+The [Chronicle Dashboard](https://chroniclelabs.org/dashboard) enables full transparency over where the Oracle-reported data was sourced and who reported/signed it.
+
+## Where is the data sourced from?
+
+Using the dashboard, users can select a time interval on the graph and, for each validator, use the drop-down arrow to see the different data sources that have been used. Typically, the data sources are a mix of on-chain and off-chain data sources with high liquidity.
+
+
+<div style={{textAlign: 'center'}}>
+<img
+    src="/img/Intro/time.png"
+    alt="Check Sources"
+    width="500"
+/>
+</div>
+<div style={{textAlign: 'center'}}>
+<img
+    src="/img/Intro/sources.png"
+    alt="Check Sources"
+    width="500"
+/>
+</div>
+In addition, users can cryptographically verify in the browser that the validator's signature has not been tampered with.
+
+<div style={{textAlign: 'center'}}>
+<img
+    src="/img/Intro/non-manipulated.png"
+    alt="Check Sources"
+    width="500"
+/>
+</div>
+
+Modifying even one character of the signature will result in invalidating the signature.
```

Comment:
Does not need to be in this first iteration, but there are two different signatures that can be verified via the dashboard.

The ECDSA signature is signed by a single validator. Its their price of the wat and allows everyone to verify the behaviour of said validator.

However, you can also click on the _Schnorr Signature_ field to verify the schnorr signature. This signature is whats actually being pushed onchain, ie it allows to verify the oracle instead of a single validator. 

I think making this distinction, verify single validator vs verify onchain price, would be beneficial re "Verifiability on every level"

Reply:
This is a great suggestion! Implemented